<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/pic1/KTLT(3).png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblNgaythue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Ngày thuê:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNgaytra">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Ngày trả:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSongaythue">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Số ngày thuê:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTongtien">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
color: rgb(78, 10, 127);</string>
    </property>
    <property name="text">
     <string>Tổng tiền:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTongtien">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ptnThanhtoan">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 5px;
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 170, 255);
</string>
    </property>
    <property name="text">
     <string>THANH TOÁN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledngaythue">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(78, 10, 127);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledngaytra">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledsongaythue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 10px;
color: rgb(255, 255, 255);
border: 1px solid rgb(78, 10, 127);
background-color: rgb(78, 10, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ptnquaylai2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
border-radius: 5px;
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Quay lại</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="giaodien2.qrc"/>
  <include location="giaodien1.qrc"/>
 </resources>
 <connections/>
</ui>
